set.mm

```diff
@@ -74348,13 +74347,34 @@ composition with a (2-place) operation to build compound operations such
   $}
 
   ${
-    $d x y z $.
+    $d x y z t $.
     $( A function that can be used to feed a common value to both operands of
        an operation.  Use as the second argument of a composition with the
        function of ~ fpar in order to build compound functions such as
-       ` y = ( ( sqrt `` x ) + ( abs `` x ) ) ` .  (Contributed by NM,
-       17-Sep-2007.) $)
+       ` ( x e. ( 0 [,) +oo ) |-> ( ( sqrt `` x ) + ( sin `` x ) ) ) ` .
+       (Contributed by NM, 17-Sep-2007.)  Replace use of ~ dfid2 with ~ df-id .
+       (Revised by BJ, 31-Dec-2023.) $)
     fsplit $p |- `' ( 1st |` _I ) = ( x e. _V |-> <. x , x >. ) $=
+      ( vy vz vt cv c1st cid wbr copab cop wceq cvv wcel wa exbii eqeq2d 3bitri
+      vex wex equsexvw cres ccnv brcnv brresi cfv 19.42v op1std eqeq1d pm5.32ri
+      cmpt wfn wb wfo fo1st fofn ax-mp fnbrfvb mp2an df-id eleq2i elopab equcom
+      ancom opeq2 pm5.32i biidd bitri 3bitrri anbi12ci 3bitr3ri opeq12d opabbii
+      anbi1i id wrel relcnv dfrel4v mpbi mptv 3eqtr4i ) AEZBEZFGUAZUBZHZABIZWBW
+      AWAJZKZABIWDALWGUJWEWHABWEWBWAWCHWBGMZWBWAFHZNZWHWAWBWCARZBRZUCGWBWAFWLUD
+      WKCEZWAKZWBWNWNJZKZNZCSZWHWBFUEZWAKZWQNZCSXAWQCSZNWSWKXAWQCUFXBWRCWQXAWOW
+      QWTWNWAWNWNWBCRZXDUGUHUIOXAWJXCWIFLUKZWBLMXAWJULLLFUMXEUNLLFUOUPWMLWBWAFU
+      QURWIWBWNDEZKZCDIZMWBWNXFJZKZXGNZDSZCSXCGXHWBCDUSUTXGCDWBVAXLWQCXLXFWNKZW
+      QNZDSWQXKXNDXKXGXJNXMXJNXNXJXGVCXGXMXJCDVBVMXMXJWQXMXIWPWBXFWNWNVDPVEQOWQ
+      WQDCXMWQVFTVGOVHVIVJWQWHCAWOWPWGWBWOWNWAWNWAWOVNZXOVKPTVGQVLWDVOWDWFKWCVP
+      ABWDVQVRABWGVSVT $.
+  $}
+
```

Comment:
Neither ~fpar nor ~fsplit are currently used in proofs. So what are they good for? The given examples are also not very enlightening.

Reply:
I haven't really considered applications. When looking at dfid2, I saw it had one discouraged use, so I just set on to remove it.